Tutorials/pinpull.md

```diff
@@ -0,0 +1,49 @@
+# Pull- Pins
+
+Suppose a pin is configured as an input. If nothing is connected to the pin and the program tries to read the state of the pin, it would be in a 'floating' state i.e an unknown state. To prevent this, a pull-up or a pull-down state is defined. They are often used in the case of Buttons and Switches.
```

Comment:
"pull-up" and "pullup" are both used in this PR, but it would be better to choose a standard. I think hyphenated is standard, but haven't researched this.